### PR TITLE
tasks: Update User guides task descrtipion for GCI 2017.

### PR DESCRIPTION
**Pull request by synicalsyntax**

### CODE REVIEW

File: tasks/2017/user-guides.md [hunk 1]
```diff
@@ -2,130 +2,99 @@
...
 
-* Good written English.
+* **Good written English.** Proper spelling, grammar, and word usage conventions
+  are critical for this task, so you should be fluent in English if you decide
+  to claim this task.
 
-* If this is your first contribution, you may be interested in the
-  [how to create a pull request](https://codein.withgoogle.com/tasks/6541581402243072/) task.
+* **Completion of the
+  [submit a pull request](../submit-a-pull-request.md) task**. The task helps
```
Comment (lonerz) [inline tasks/2017/user-guides.md]: Shouldn't this be `./submit-a-pull-request.md`?

File: tasks/2017/user-guides.md [hunk 2]
```diff
@@ -2,130 +2,99 @@
...
-## General Notes
-
-* The source for the user documentation is the Markdown files under
-  `templates/zerver/help/` in the main Zulip repository (zulip/zulip).
-
-* To see the current, Markdown version of a user documentation file, reload
-  your browser on `http://<hostname>:9991/help/<feature>`, where
-  `<hostname>` is either `localhost` or the hostname of your VM, and
-  `<feature>` is the name of the feature.
+Well-written user documentation help users and search engines discover Zulip
```
Comment (lonerz) [inline tasks/2017/user-guides.md]: r/help/helps

**Comment by timabbott** [comment]
Looks great, merged!  I think we might want to resurrect a variant of "task type B" where their job is to go through a group of 5 articles, try to follow the instructions somewhat blindly, taking notes on what didn't work.  And then also compare with the UI and see what other articles should possibly be link.  And the work product is a list of things about that documentation that can be improved.

I think we could have ~20 tasks that way (by grouping articles; I think we want each to involves carefully reviewing ~3-5 pages of text).  

@synicalsyntax also, we should produce a script with a list of articles to write.  I'd shy away from the "document app X" articles that didn't happen last year for early tasks, since I think the reason they didn't happen is they're larger/more difficult projects, and we want to start with 20+ user docs tasks that are easy to do.

**Comment by timabbott** [comment]
Opened https://github.com/zulip/zulip-gci/issues/635 for the new task.

**Pull request merged**